NOTICE.txt

```diff
@@ -21,7 +342,1033 @@ Launch node applications or executables at login (Mac, Windows, and Linux)
+
+---
+
+## eslint-plugin-react
+
+This product contains 'eslint-plugin-react' by Yannick Croissant.
+
+React specific linting rules for ESLint
+
+* HOMEPAGE:
+  * https://github.com/yannickcr/eslint-plugin-react
+
+* LICENSE: MIT
+
+The MIT License (MIT)
+
+Copyright (c) 2014 Yannick Croissant
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+---
+
+## file-loader
+
+This product contains 'file-loader' by webpack-contrib.
+
+The file-loader resolves import/require() on a file into a url and emits the file into the output directory.
+
+* HOMEPAGE:
+  * https://github.com/webpack-contrib/file-loader
+
+* LICENSE: MIT
+
+Copyright JS Foundation and other contributors
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+'Software'), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+---
+
+## font-awesome
+
+This product contains 'font-awesome' by Fort Awesome.
+
+The iconic SVG, font, and CSS framework
+
+* HOMEPAGE:
+  * https://github.com/FortAwesome/Font-Awesome
+
+* LICENSE: Font Awesome Free License
+-------------------------
```

Comment:
If this is the correct one, then we need to contact the repo owners because the license is not one of the open source ones https://opensource.org/licenses/alphabetical.

Reply:
which one isn't open source? there is a different license depeding on the asset (src, icons or fonts) but I believe all of them are open.

Reply:
I see now that there are different licenses listed.